TROUBLESHOOTING.md

```diff
@@ -11,6 +11,7 @@ This guide helps you diagnose and resolve common issues with the Azure MCP Serve
     - [Can I select what tools to load in the MCP server?](#can-i-select-what-tools-to-load-in-the-mcp-server)
     - [Why does VS Code only show a subset of tools available?](#why-does-vs-code-only-show-a-subset-of-tools-available)
     - [VS Code Permission Dialog for Language Model Calls](#vs-code-permission-dialog-for-language-model-calls)
+    - [Problems with VS Code Cache](#problems-with-vs-code-cache)
```

Comment:
Can you reword this so "VS Code" starts the sentence. Eg "VS Code Cache Problems"